### ISSUE TITLE
zoom via mouse wheel on terrain style

### ISSUE THREAD

**Issue by d-chambers** [pyvista/pyvista#1022]
## Description

Hello,

I find that the terrain style of camera control is much more intuitive for the types of data I am plotting. However, after calling the `enable_terrain_style` method on the `Plotter` instance I can no longer use the mouse wheel for zooming in and out. Is there a manual way to enable zooming while maintaining the "Z is up" awareness offered by the terrain style?

Thanks!

## Example Data

```python
import pyvista as pv
from pyvista import examples

mesh = examples.download_st_helens().warp_by_scalar()

plotter = pv.Plotter()
plotter.add_mesh(mesh)
plotter.enable_terrain_style()
plotter.show()
# plotter.enable_zoom()  # not a real method...

```

**Comment by banesullivan** [comment]
We should provide a way to easily define a custom interactor. vtk.js has a really nice version of this that we could mimic: https://kitware.github.io/vtk-js/examples/InteractorStyleManipulator.html and https://discourse.vtk.org/t/vtk-js-trackball-default-interaction-style-with-pan-zoom-on-other-mouse-buttons/4059/4?u=banesullivan

**Comment by adeak** [comment]
@d-chambers we've added this in https://github.com/pyvista/pyvista/pull/1376 and the new feature should be in the next PyVista release.

You'll be enable to use
```py
plotter.enable_terrain_style(mouse_wheel_zooms=True)
```

to re-enable mouse wheel zooming in terrain style. Similarly,
```py
plotter.enable_terrain_style(shift_pans=True)
```

to make shift+left mouse pan.

**Comment by d-chambers** [comment]
@adeak thanks, this will be very useful!

**Comment by d-chambers** [comment]
closed by #1376
